src/components/AppNavigation/SettingsSection.vue

```diff
@@ -23,7 +23,7 @@
 
 <template>
 	<div>
-		<div v-if="allowSocialSync">
+		<div v-if="allowSocialSync" class="socialSyncListEntry">
```

Comment:
Should use [BEM naming style](https://cssguidelin.es/#bem-like-naming)

```suggestion
 		<div v-if="allowSocialSync" class="social-sync__list-entry">
```

Reply:
done